ci/gpu/build.sh

```diff
@@ -61,6 +61,11 @@ gpuci_mamba_retry install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvid
       "rapids-doc-env=${MINOR_VERSION}.*" \
       "shap>=0.37,<=0.39"
 
+gpuci_mamba_retry remove -c conda-forge -c rapidsai -c rapidsai-nightly -c nvidia \
```

Comment:
Can you use `gpuci_conda_retry remove` instead of `mamba` here? It should fix CI issues

Reply:
Thanks!